infrastructure/transform/main.tf

```diff
@@ -65,6 +65,45 @@ resource "azurerm_data_factory" "adf" {
   }
 }
 
+
+resource "azurerm_data_factory_pipeline" "pipeline" {
+  name            = "databricks-pipeline-${var.naming_suffix}"
+  data_factory_id = azurerm_data_factory.adf.id
+  activities_json = <<JSON
+[
+  {
+    "name": "DatabricksPythonActivity",
+    "type": "DatabricksSparkPython",
+    "typeProperties": {
+        "pythonFile": "dbfs:/entrypoint.py",
+        "libraries": [
+            {
+                "whl": "dbfs:/src-0.0.1-py3-none-any.whl"
+            }
+        ]
+    },
+    "linkedServiceName": {
+        "referenceName": "ADBLinkedServiceViaMSI",
+        "type": "LinkedServiceReference"
+    }
+  }
+]
+  JSON
+}
+
+resource "databricks_dbfs_file" "dbfs_file_upload" {
+  source = "${var.whl_file_local_path}/${var.whl_file_name}"
+  path   = "/${var.whl_file_name}"
+}
+
+resource "azurerm_data_factory_trigger_schedule" "trigger" {
+  name            = "databricks-pipeline-trigger-${var.naming_suffix}"
+  data_factory_id = azurerm_data_factory.adf.id
+  pipeline_name   = azurerm_data_factory_pipeline.pipeline.name
+  interval        = 5
```

Comment:
Let's make it 15 for now, we can always shorten it down the line

Reply:
Done